<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="pageLogin">
       <widget class="QWidget" name="formContainer&#10;" native="true">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>30</y>
          <width>561</width>
          <height>473</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>15</number>
         </property>
         <property name="leftMargin">
          <number>30</number>
         </property>
         <property name="topMargin">
          <number>30</number>
         </property>
         <property name="rightMargin">
          <number>30</number>
         </property>
         <property name="bottomMargin">
          <number>30</number>
         </property>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Login to Plota Game </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="formFields&#10;" native="true">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Username :</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Password :</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLineEdit" name="txtPassword">
              <property name="text">
               <string/>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::EchoMode::Password</enum>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QPushButton" name="btnLogin">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                <horstretch>35</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
    background-color: #2E86C1;
    color: white;
    border-radius: 6px;
}
QPushButton:hover {
    background-color: #1B4F72;
}
</string>
              </property>
              <property name="text">
               <string>Login</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="txtUsername">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string> </string>
              </property>
              <property name="placeholderText">
               <string/>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QPushButton" name="btnSignUp">
              <property name="styleSheet">
               <string notr="true">QPushButton {
    background: transparent;
    color: #2E86C1;
    border: none;
}
QPushButton:hover {
    text-decoration: underline;
}
</string>
              </property>
              <property name="text">
               <string>Sign Up</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QPushButton" name="btnForgot">
              <property name="styleSheet">
               <string notr="true">QPushButton {
    background: transparent;
    color: #2E86C1;
    border: none;
}
QPushButton:hover {
    text-decoration: underline;
}
</string>
              </property>
              <property name="text">
               <string>Forgot Password</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="pageMainMenu">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="menuContainer&#10;" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>40</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="5" column="0">
            <widget class="QPushButton" name="btnLogout">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #34495E;
    color: white;
    border-radius: 8px;
    font-size: 12pt;
}
QPushButton:hover {
    background-color: #1ABC9C;
}

</string>
             </property>
             <property name="text">
              <string>Logout</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <pointsize>18</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Main Menu</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="btnStartGame">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #34495E;
    color: white;
    border-radius: 8px;
    font-size: 12pt;
}
QPushButton:hover {
    background-color: #1ABC9C;
}

</string>
             </property>
             <property name="text">
              <string>Start Game</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="btnHistory">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #34495E;
    color: white;
    border-radius: 8px;
    font-size: 12pt;
}
QPushButton:hover {
    background-color: #1ABC9C;
}

</string>
             </property>
             <property name="text">
              <string>Game History</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QPushButton" name="btnProfile">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #34495E;
    color: white;
    border-radius: 8px;
    font-size: 12pt;
}
QPushButton:hover {
    background-color: #1ABC9C;
}

</string>
             </property>
             <property name="text">
              <string>Profile</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lblWelcome&#10;">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Welcome Player</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageGameSelection">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="2" column="0">
            <widget class="QPushButton" name="btnConnectFour">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #34495E;
    color: white;
    border-radius: 8px;
    font-size: 12pt;
}
QPushButton:hover {
    background-color: #1ABC9C;
}
</string>
             </property>
             <property name="text">
              <string>ConnectFour</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QPushButton" name="btnBack">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #34495E;
    color: white;
    border-radius: 8px;
    font-size: 12pt;
}
QPushButton:hover {
    background-color: #1ABC9C;
}
</string>
             </property>
             <property name="text">
              <string>Back</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <spacer name="verticalSpacer_7">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Select a Game</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="btnOthello">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #34495E;
    color: white;
    border-radius: 8px;
    font-size: 12pt;
}
QPushButton:hover {
    background-color: #1ABC9C;
}
</string>
             </property>
             <property name="text">
              <string>Othello</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <spacer name="verticalSpacer_8">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageConnectFourBoard">
       <property name="styleSheet">
        <string notr="true">QWidget#pageConnectFourBoard {
    background-color: qradialgradient(
        cx:0.5, cy:0.4, radius:1.2,
        stop:0 #1b2735,
        stop:0.4 #090a0f,
        stop:1 #000000
    );
}

QWidget#connectFourContainer {
    background-color: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #141e30,
        stop:1 #243b55
    );
    border-radius: 30px;
    border: 3px solid #00eaff;
    padding: 20px;
}

QPushButton {
    background-color: #0f2027;
    border-radius: 36px;
    border: 2px solid #00eaff;
}

QPushButton:hover {
    border: 2px solid #ffffff;
}

QPushButton#btnBackFromConnectFour {
    background-color: #111;
    color: #00eaff;
    border: 2px solid #00eaff;
    border-radius: 15px;
    padding: 8px 18px;
    font-weight: bold;
}

QPushButton#btnBackFromConnectFour:hover {
    background-color: #00eaff;
    color: black;
}

QLabel#label_6 {
    background-color: #111;
    color: #00eaff;
    border: 2px solid #00eaff;
    border-radius: 15px;
    padding: 8px 18px;
    font-weight: bold;
}

</string>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>911</width>
          <height>451</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <pointsize>24</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::FocusPolicy::WheelFocus</enum>
           </property>
           <property name="text">
            <string>Connect Four</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QWidget" name="connectFourContainer" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="btnBackFromConnectFour">
           <property name="styleSheet">
            <string notr="true">

</string>
           </property>
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>956</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
